benefit/data/sheets/CaseField/CaseField.json

```diff
@@ -407,5 +407,7 @@
   {"LiveFrom": "20/08/2019", "CaseTypeID": "Benefit", "ID": "summaryConfidentialityRequired", "Label": "**Confidentiality Required**  \r\n${appeal.appellant.confidentialityRequired}", "FieldType": "Label", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "01/11/2021", "CaseTypeID": "Benefit", "ID": "isFqpmRequired", "Label": "Case requires a Financially Qualified Panel Member (FQPM)", "FieldType": "YesOrNo", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "01/11/2021", "CaseTypeID": "Benefit", "ID": "otherPartyHearingRecordingReq", "Label": "Other party hearing recording requests", "FieldType": "Collection", "FieldTypeParameter": "hearingRecordingRequest", "SecurityClassification": "PUBLIC"},
-  {"LiveFrom": "01/11/2021", "CaseTypeID": "Benefit", "ID": "otherPartyHearingRecordingReqUi", "Label": "Other party hearing recording requests", "FieldType": "Collection", "FieldTypeParameter": "otherPartyHearingRecordingReqUi", "SecurityClassification": "PUBLIC"}
+  {"LiveFrom": "01/11/2021", "CaseTypeID": "Benefit", "ID": "otherPartyHearingRecordingReqUi", "Label": "Other party hearing recording requests", "FieldType": "Collection", "FieldTypeParameter": "otherPartyHearingRecordingReqUi", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "01/12/2021", "CaseTypeID": "Benefit", "ID": "otherPartyOptions", "Label": "Resend to other party", "FieldType": "Collection", "FieldTypeParameter": "OtherPartyOption", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "01/12/2021", "CaseTypeID": "Benefit", "ID": "showReissueToOtherPartyUiSection", "Label": "Show Reissue To Other Party Ui Section", "FieldType": "YesOrNo", "SecurityClassification": "PUBLIC"}
```

Comment:
Label text should have initial cap to be consistent with other labels.

Reply:
I'll fix that but that field will never be displayed.